configs/tiny/OLMo-300M.yaml

```diff
@@ -133,31 +139,19 @@ evaluators:
 
   - label: openbook_qa
     type: downstream
-
-  - label: boolq
-    type: downstream
  
   - label: sciq
     type: downstream
 
   - label: arc_easy
     type: downstream
 
-  - label: arc_challenge
-    type: downstream
-
-  - label: copa
-    type: downstream
-
   - label: commonsense_qa
     type: downstream
 
   - label: social_iqa
     type: downstream
 
-  - label: basic_arithmetic
-    type: downstream
-
```

Comment:
What's wrong with these?

Reply:
ah, ``basic_arithmetic`` should be in, others don't provide any signal based on my experience

Reply:
ah this was commented out saying

``# Doesn't work from cache.``

Reply:
Should work with cache v4